src/app/media/media.component.ts

```diff
@@ -71,7 +71,7 @@ export class MediaComponent {
       });
       this.selectedMedia=true;
     }
-    if(mediaType="show")
+    else if(mediaType="show")
```

Comment:
This line and line 66 need changed to be a double or tripple =. (mediaType=="show") and (mediaType=="movie") respectively. As is it its setting mediaType to show/media not checking to see if mediaType is actually a show or movie

Reply:
Will chang sit soon